### DISCUSSION TITLE
Ubuntu 18.04 - 20.04 (and soon 22.04)

### DISCUSSION THREAD

**Discussion by vermaete** [OpenDDS/OpenDDS#3462] [General]
Hi,

The [documenation ](https://opendds.org/documents/building.html) is about Ubuntu 18.
But in the [CI/CD](https://github.com/objectcomputing/OpenDDS/blob/master/.github/workflows/build_and_test.yml#L11320) there seems to be Ubuntu 20.04 too.  Besides Ubuntu 18.04 that will be kind of end of life next year.

Is it just the documentation that  should be updated?  Of is  Ubuntu 18.04 still more tested than 20.04?

br

**Comment by mitza-oci** [comment]
The documentation list the platforms that we run the entire test suite on.  We will eventually move that Ubuntu 18 VM up to the 20 or 22 LTS, but it hasn't happened yet. 18 is still widely used in the field.
